addons/forum-user-agent/addon.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Forum user-agent Modifier",
```

Comment:
```suggestion
  "name": "Modify user-agent slightly in forums",
```

Reply:
It's hard to come up with a good name, but I think it makes sense to mention the forums, and it is also better if we emphasize the fact the addon only does a very "subtle" or "small" change.

Reply:
I think something like "Modify browser version [on forums](https://scratchaddons.com/docs/reference/addon-manifest/#description:~:text=on%20(not%20in)%20the%20forums)" or "Modify version information on forums" would be better because we don't actually modify the browser's user agent string. The description could say "globe icon" instead of "user agent button".

Reply:
> we don't actually modify the browser's user agent string

I personally don't think this matters. Most people don't know what the user-agent string is anyway.
Those who know, after reading the description, will understand that the addon only affects the output of the globe icon.

Still, `Modify version information on forums` is okay, the addon name is boring and it's quite clear that it does nothing interesting.